spec/unit/validator_spec.rb

```diff
@@ -907,6 +907,8 @@ def test_swiftpod
 
         Podfile::TargetDefinition.any_instance.stubs(:uses_frameworks?).returns(true)
         Pod::Sandbox::FileAccessor.any_instance.stubs(:source_files).returns([pathname])
+        Pod::Installer::PodSourceInstaller.any_instance.stubs(:lock_files!)
+        Pod::Installer::PodSourceInstaller.any_instance.stubs(:unlock_files!)
```

Comment:
My changes broke these tests, I think because of the way `/Foo.swift` was stubbed into `:source_files` here. Not really sure why that broke it though

Reply:
would be nice to figure out

Reply:
I can't seem to figure it out. If its a deal breaker I can simply undo the change to `PodSourceInstaller` that caused it